src/ResourceParser.h

```diff
@@ -49,21 +52,15 @@ namespace snowcrash {
 
                 // Make this section an action
                 if (!captureGroups[2].empty()) {
+                    return processNestedAction(node, siblings, pd, layout, out);
```

Comment:
I don't think the `siblings` given here is correct. If you look at the old code, they are `node->parent().children()`